### PR TITLE
cpp: Remove remaining references of `dd-opentracing-cpp`

### PR DESCRIPTION
**Pull request by dmehala**
### What does this PR do? What is the motivation?

This PR removes remaining references of `dd-opentracing-cpp` in fr and en. 

### Merge instructions

- [x] Please merge after reviewing

### Additional notes
I noticed that the C++ documentation translated into Japanese and Korean hasn't been updated. What is the process for translating these documents?

### CODE REVIEW

File: content/fr/tracing/guide/setting_up_APM_with_cpp.md [hunk 1]
```diff
@@ -8,21 +8,21 @@ further_reading:
...
 
-Ce guide vient compléter la [documentation sur la configuration de l'APM en C++][1]. Les instructions qui y sont détaillées vous permettent de configurer une application simple en C++ avec l'APM sur votre VM, afin de diagnostiquer vos éventuels problèmes.
+Ce guide vient compléter la [documentation sur la configuration du traceur APM en C++][1]. Les instructions qui y sont détaillées vous permettent de configurer une application simple en C++ avec APM sur votre VM, afin de diagnostiquer vos éventuels problèmes.
 
 ## Configuration de votre solution
 
 ### Environnement de base
 
-Commencez par lancer une nouvelle box Vagrant `ubuntu/xenial64` et y exécuter la commande `ssh` :
+Commencez par lancer une nouvelle box Vagrant `ubuntu/jammy64` et connectez vous a la VM :
```
Comment (dmehala) [inline content/fr/tracing/guide/setting_up_APM_with_cpp.md:17]:
```suggestion
Commencez par lancer une nouvelle box Vagrant `ubuntu/jammy64` et connectez vous à la VM :
```

File: content/en/tracing/trace_collection/proxy_setup/_index.md [hunk 2]
```diff
@@ -279,22 +278,15 @@ To configure your sampling rate with `DD_TRACE_SAMPLING_RULES`, use one of the f
 
 ## Environment variables
 
-The available [environment variables][3] depend on the version of the C++ tracer embedded in Envoy.
-
-**Note**: The variables `DD_AGENT_HOST`, `DD_TRACE_AGENT_PORT` and `DD_TRACE_AGENT_URL` do not apply to Envoy, as the address of the Datadog Agent is configured using the `cluster` settings.
+<div class="alert alert-warning">
+  <strong>Note:</strong> The variables `DD_AGENT_HOST`, `DD_TRACE_AGENT_PORT` and `DD_TRACE_AGENT_URL` do not apply to Envoy, as the address of the Datadog Agent is configured using the `cluster` settings.
```
Comment (rtrieu) [inline content/en/tracing/trace_collection/proxy_setup/_index.md:282]:
```suggestion
  <strong>Note:</strong> The variables <code>DD_AGENT_HOST</code>, <code>DD_TRACE_AGENT_PORT</code> and <code>DD_TRACE_AGENT_URL</code> do not apply to Envoy, as the address of the Datadog Agent is configured using the <code>cluster</code> settings.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Hi @dmehala, thank you for your PR! I made one suggestion so that the variables appear as code instead of being enclosed by backticks. Thank you for editing the French version, but our translators will handle it so I would ask you to discard your changes in /fr/, because they'll be overwritten by our Translation pipeline anyway. To answer your question, any change made to the English version gets translated by Transifex a few days or weeks after, but sometimes they get a backlog. I will ask them to prioritize these files in Japanese and Korean!

**Comment by rtrieu** [comment]
@dmehala  FYI, I have prioritized the translation of this file.

Screenshot 2024-05-21 at 10 39 42 AM

**Review by rtrieu — APPROVED** [review]
Looks good, thank you for these updates!

**Pull request merged**